src/edge_index.cpp

```diff
@@ -177,27 +173,21 @@ void _writeIndexGroup(const std::vector<NodeID>& nodeIDs, uint64_t nodeCount, Hi
             auto& ranges = it->second;
             primaryIndex.push_back({offset, offset + ranges.size()});
             offset += ranges.size();
-            std::move(
-                ranges.begin(), ranges.end(),
-                std::back_inserter(secondaryIndex)
-            );
+            std::move(ranges.begin(), ranges.end(), std::back_inserter(secondaryIndex));
```

Comment:
I don't understand what this instruction is doing. Honestly, I don't understand why it compiles. Did you mean `std::copy` instead?

Reply:
As I mentioned in the previous comment, this is not from the reports.
My code is under reports folder.

Maybe I should create the PR upstream directly to show only my changes...